src/errors.js

```diff
@@ -15,69 +15,44 @@ const inherits = require('inherits');
  *
  * @param {string} msg error message
  */
-function BaseError(msg) {
-  BaseError.super_.call(this, msg);
-  if(Error.captureStackTrace) {
-    Error.captureStackTrace(this, this.constructor);
+function BaseError (msg) {
+  BaseError.super_.call(this, msg)
```

Comment:
`super(msg)` should do the trick

Reply:
we can, but I'd prefer a separate PR for that